addons/mail/static/src/components/messaging_menu/messaging_menu.scss

```diff
@@ -50,6 +50,7 @@
 
     &:not(.o-mobile) {
         display: flex;
+        flex: 1 0 auto;
```

Comment:
I don't think this should be limited here.
The goal is to prevent the shrink that is coming from the display: flex on the parent, which is applied in all cases I think (mobile or not).
The flex on self should not matter in this regard.

Reply:
Safari "strangely" shrinks children in flex, and it's just in some weird cases. Right now, I just see this item, so the rest is good. Preventing flex children to shrink fixes the issue (`flex-shrink: 0`, btw I prefer this explicit ruleset than `flex: 1 0 auto` because I can't remember whether `flex-shink:` is 1st or 2nd parameter of `flex:`...).

When doing some research, Safari justifies strange behaviour because flex specs make sense when flex container has provided `height: ` (not `auto`, ofc). Thing is, we don't want to provide non-auto `height` because this kills purpose of letting content define the size, and we prefer defining thresholds with `min-height` and `max-height`.

IMO, Chrome and Firefox make total sense even though it's not in flex specs, but Safari justifying doing stupid things when something doesn't follow the rule... No common sense I guess...

So we can't follow the "flex" rules that Safari wants, unfortunately, but the `flex-shrink: 0` makes Safari behaves kindly as Chrome and Firefox do.

Reply:
So even without current PR, the mobile version is actually fine for this error?

Reply:
Don't see any problem in mobile without this PR.

Reply:
Ok well feel free to merge then.

I guess the problem can be limited to "flex inside flex when the parent (?) doesn't have a specific height"?

Reply:
> I guess the problem can be limited to "flex inside flex when the parent (?) doesn't have a specific height"?

Which is basically almost everything:
- everything should be flex (and sometimes grid)
- everything should have size based on content (few exceptions which have to auto-grow with available space, or established max with overflow management)